pkgs/development/python-modules/maestral/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchFromGitHub
+, python
+, sdnotify
+, systemd
+}:
```

Comment:
```suggestion
}:

```

Reply:
Done. Also added the whitespace for the maestral-qt package.